include/eve/module/core/regular/bit_ceil.hpp

```diff
@@ -97,15 +98,19 @@ namespace eve
         e           = dec(e);
         auto tmp    = ldexp(one(eve::as(v)), e);
         auto tmpltv = tmp < v;
-        if constexpr( scalar_value<T> ) { return tmpltv ? tmp + tmp : tmp; }
-        else { return if_else(vle1, one(eve::as(v)), if_else(tmpltv, tmp + tmp, tmp)); }
+        if constexpr( scalar_value<T> )
+          return tmpltv ? tmp + tmp : tmp;
```

Comment:
the carriage returns were uneccessary

Reply:
ok, but I prefer this kind of indentation